torch/_inductor/decomposition.py

```diff
@@ -35,7 +37,17 @@
         aten.unsafe_split,
     ]
 )
-decompositions = {**core_aten_decompositions(), **inductor_decompositions}
+quantized_decomps = get_decompositions(
```

Comment:
`get_decompositions()` looks up an existing decomposition registered in `torch._decomp`.  Are there registered decomps for these there? If not I think this will just return an empty dict.

Reply:
No, it's not registered there. Remove `quantized_decomps`. Please help to take a check.